frontend-panel/src/index.css

```
@@ -297,33 +297,85 @@
-[data-sonner-toast][data-x-position="right"][data-y-position="bottom"] {
-	--toast-slide-x: 110%;
-	transform: var(--y) translateX(var(--toast-slide-x));
-	transition:
-		transform 220ms ease,
-		opacity 220ms ease,
-		height 400ms,
-		box-shadow 200ms;
+[data-sonner-toaster] {
+	--width: min(24rem, calc(100vw - 1.5rem));
+	--border-radius: min(var(--radius-lg), 0.625rem);
+	font-family: var(--font-sans);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)